projects/deno.land/package.yml

```diff
@@ -18,13 +18,18 @@ runtime:
     DENO_NO_UPDATE_CHECK: 'true'
     DENORT_BIN: '{{prefix}}/bin/denort'
 
-dependencies:
-  darwin/x86-64:
-    # FIXME try removing this after new builds are available
-    llvm.org: 17 # libunwind
-
 build:
   script:
+    # deno does not need llvm to build on darwin+x86-64, and if it is present,
+    # deno will be linked to its libunwind, which then causes deno to need llvm
+    # in runtime.
+    - |
+      if test "{{hw.platform}}+{{ hw.arch }}" != "darwin+x86-64"; then
+        set -o allexport
+        source <(pkgx +llvm.org^17)
+        set +o allexport
+      fi
+
```

Comment:
so, shouldn't this be a build depedency?
```
build:
  dependency:
    linux:
      llvm.org: ^17
    darwin/aarch64:
      llvm.org: ^17
```
?

Reply:
For some weird reason it doesn't work because of:

> For some reason when adding rust later, llvm also needs to be added later.

_Originally posted by @felipecrs in https://github.com/pkgxdev/pantry/issues/5751#issuecomment-2029063223_

Reply:
But let's try once again: https://github.com/pkgxdev/pantry/pull/5765/commits/bcd5ec3a122ff664a340d69e8db0fd79cfd7c241

Reply:
Yeah:

```
configure: error: no acceptable ld found in $PATH
```

https://github.com/pkgxdev/pantry/actions/runs/8515078419/job/23321905009?pr=5765#step:6:1399